### ISSUE TITLE
Migration to crowdin to weblate

### ISSUE THREAD

**Issue by gutierri** [GnuCash-Pocket/gnucash-android#3]
As part of the resumption of the project, we need a new translation platform for this branch. Bringing the discussion of issue #913 to weblate seems to be more appropriate and easy to use.

I wonder what translators think about. If relevant, we will create an org there to keep the new translations.

**Comment by djbrown** [comment]
@comradekingu can you provide a quick overview of the benefits when moving to weblate?
Also: how complex and time consuming would the migration be? which components besides the weblate service have to be updated? i think at least the build, and maybe other configuration?

We can request access to the crowdin project via support, which could make the migration easier (export/import).
Also the project could keep using crowdin until the migration is finished.
@pedroabel @pbneves  what do you think?

**Comment by lxbzmy** [comment]
I find (bing) a discuss of  translation platform: https://github.com/mumble-voip/mumble/discussions/4641

**Comment by djbrown** [comment]
another big argument is, that GnuCash already uses Weblate too.
a future translation integration would certainly be easier if GnuCash Pocket were on Weblate too.

but how about the free quota, are they acceptable, or could they soon be exhausted by this project?
> max 10,000 source strings
> max 60 target languages

**Comment by comradekingu** [comment]
@djbrown Can't use that account.
Weblate has a better overview of strings, and triggered checks to correct.

**Comment by djbrown** [comment]
@comradekingu yes, separate free account. I mean some day, when the GnuCash Pocket project has poven itself, then maybe it could also integrate more with GnuCash. migrating to weblate (even a separate account), would pave the way for future synergy in the translations area.

**Comment by comradekingu** [comment]
Do you want me to just set up a seperate GnuCash Pocket project on Hosted?

**Comment by djbrown** [comment]
would that be possible with the free tier? we don't have budget for paid services 😬

**Comment by Kaligula0** [comment]
I see there's a free Libre Plan:

![SmartSelect_20231222_115624_Chrome](https://github.com/GnuCash-Pocket/gnucash-android/assets/3950811/19c151dc-2b92-4657-a42f-7a0a2c707ef4)
